src/plugins/discover/public/embeddable/saved_search_embeddable.tsx

```diff
@@ -266,7 +267,8 @@ export class SavedSearchEmbeddable
       this.searchProps!.totalHitCount = resp.hits.total as number;
       this.searchProps!.isLoading = false;
     } catch (error) {
-      if (!this.destroyed) {
+      const cancelled = !!prevAbortController?.signal.aborted;
+      if (!this.destroyed && !cancelled) {
```

Comment:
Could you please check one case? I don't see an error message after deleting a saved search. My steps were:
- Install sample data
- Create a new dashboard and add a saved search to it
- Refresh multiple times <= all good, no Abort message
- Delete that saved via Stack Management which was opened in another tab
- Press "Refresh" => weird, Dashboard still shows the table for some reason
- Refresh the browser page <= All good, the expected error is shown this time

Reply:
Yes, but it's how it worked before. In this case, we are not breaking anything.

Reply:
We synced and decided it's not a blocker but an issue should be added, it may worth to check if this is also the behavior when using other embeddables, I'd guess so, since the embeddables were loaded, and I guess they won't check if they still exist

Reply:
This thing happens with other saved searches on dashboard, but also on main discover view and lens. Seems like it’s general behaviour in kibana.